prisma/schema.prisma

```diff
@@ -9,3 +6,28 @@ datasource db {
   provider = "postgresql"
   url      = env("DATABASE_URL")
 }
+
+model City {
```

Comment:
Some platforms are case-sensitve. Try to patternize your models with lowercase like that: 
eg: @@map("city")

Reply:
resolved with recent commit